source/LumiCalReco/src/LumiCalClusterer_getCalHits.cpp

```diff
@@ -78,7 +78,9 @@ int LumiCalClustererClass::getCalHits(	EVENT::LCEvent * evt,
 
       } else {
 	arm =(*_mydecoder)( calHitIn )["barrel"]; // from 1 and 2
-	phiCell = (*_mydecoder)( calHitIn )["phi"];
+	//FIXME: add check for phi range?
+	phiCell = (*_mydecoder)( calHitIn )["phi"]; // goes from -phiMax/2 to +phiMax/2
+	if(phiCell < 0 ) phiCell += _cellPhiMax; // need to put into positive range only
```

Comment:
Does it really fix problem ? Is phiCell = 0 within possible values ? Historically phiCell run from 0 -> 47
and now ( -24 -> 24 ) including 0 ? Something is fishy.